src/Service.java

```diff
@@ -1,5 +1,16 @@
 public class Service {
-    String name;
-    double price;
-    int durationInMinutes;
+    private static int nextId = 0;
+
+    private int id;
+    private String name;
+    private double price;
+    private int durationInMinutes;
+
```

Comment:
there should be constructor which will set `this.id = Service.nextId++;` because now id will always be set to 0 (default value for "int" fields)